pkgs/development/tools/misc/gdb/default.nix

```diff
@@ -92,6 +92,7 @@ stdenv.mkDerivation rec {
     "--disable-shared" "--enable-static"
     "--with-system-zlib"
     "--with-system-readline"
+    "--disable-sim"
```

Comment:
Perhaps just updating `gdb` is less drastic than disabling `sim` entirely: https://github.com/NixOS/nixpkgs/pull/155233
(and while at it apply https://sourceware.org/pipermail/binutils/2022-February/119742.html)

Reply:
In fact I can now compile `gdb` on this branch without this patch.... I'll push the change with the next batch of changes :)